thumbor/engines/__init__.py

```diff
@@ -129,6 +129,8 @@ def get_mimetype(cls, buffer):
             img_mime = "image/jp2"
         elif buffer[4:12] in (b"ftypavif", b"ftypavis"):
             img_mime = "image/avif"
+        elif buffer[4:12] in (b"ftypheic", b"ftypmif1"):
```

Comment:
Hey @bigcat88, Why you didn't validate other magic number for `HEIC/HEIF`? Can you explain to me, please?!

Like [here](https://github.com/bigcat88/pillow_heif/blob/3ffc497617da6c45a1bbaf7cf59e4107ac047df6/pillow_heif/as_opener.py#L120) 

```python
if magic[8:12] in (
        b"heic",
        b"heix",
        b"heim",
        b"heis",
        b"hevc",
        b"hevx",
        b"hevm",
        b"hevs",
        b"mif1",
        b"msf1",
    ):
```

Reply:
Maybe `heix` can be added, it is a `10 bit` Canon cameras format with `.hif` extension.
I never seen other identifiers and not sure that libheif will be able to parse them correctly.
`hevc`, `hevx`, `hevm`, `hevs` is a video formats - for now it is better to skip them.

Reply:
Nice! Thanks for the explanation, and the updates!